### PR TITLE
Add Decoding Flutter videos to API docs

### PR DESCRIPTION
**Pull request by tvolkert**
## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

### CODE REVIEW

File: packages/flutter/lib/src/widgets/media_query.dart [hunk 1]
```diff
@@ -234,6 +234,8 @@ class MediaQueryData {
   ///
   /// Padding is derived from the values of [viewInsets] and [viewPadding].
   ///
+  /// {@youtube 560 315 https://www.youtube.com/watch?v=ceCo8U0XHqw}
```
Comment (Piinks) [inline packages/flutter/lib/src/widgets/media_query.dart:237]: Can you add this above on viewInsets as well?

File: packages/flutter/lib/src/widgets/primary_scroll_controller.dart [hunk 2]
```diff
@@ -17,6 +17,8 @@ const Set<TargetPlatform> _kMobilePlatforms = <TargetPlatform>{
 
 /// Associates a [ScrollController] with a subtree.
 ///
+/// {@youtube 560 315 https://www.youtube.com/watch?v=33_0ABjFJUU}
```
Comment (Piinks) [inline packages/flutter/lib/src/widgets/primary_scroll_controller.dart:20]: Can you add this one to ScrollView.primary as well? It's templated for every subclass with primary.

**Review by Piinks — COMMENTED** [review]
Thanks! I ended up getting side tracked watching these!

**Comment by tvolkert** [comment]
Comments addressed - PTAL 🙂

**Review by Piinks — APPROVED** [review]
Flutter_LGTM

**Pull request merged**